research/slim/nets/mobilenet/mobilenet_example.ipynb

```diff
@@ -197,9 +197,10 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    with tf.contrib.slim.arg_scope(mobilenet_v2.training_scope(is_training=False)):
</pre><p>should be slim.arg_scope (not tf.contrib.slim.arg_scope)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/models/pull/10182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>